### ISSUE TITLE
[BUG] App freezes and phone reboots.

### ISSUE THREAD

**Issue by WoaDmulL** [ananjaser1211/hadesRom_Reports#61]
**Describe the bug**
Normal usage of any app that freezes and after ~30s the phone reboots.

**To Reproduce**
1. Open app, use it, probably it has to do with wifi or RAM usage.
2. Nothing special, just wait for it to freeze, the browser for example, I was reading the issues searching for my specific issue and the page froze and the phone restarted.

**Smartphone (please complete the following information):**
 - Device: Note 8
 - Model: N850F

**Additional context**
I'll add logs if instructed to do so, also I've rooted the phone so I can provide mostly anything if it helps.

**Comment by ananjaser1211** [comment]
well this is a first, we had freeze reports on S8/S8+ only (including from testers) but never an N8. not much to go on. use logging kit and the guide in the main page of this repo to take logs

i need `log_all` and for you to copy `/data/tombstones`

**Comment by WoaDmulL** [comment]
The tombstones: https://gofile.io/d/BTBppo
I'll upload the logcat later after it finishes, it's still outputting.

Edit: It also seem to reboot randomly without actually being used, it was around me and saw the hades boot logo animation and when it booted the GSM wasn't working until I put the phone in airplane mode and disabled it.

**Comment by WoaDmulL** [comment]
Log_all: https://gofile.io/d/4MwUli

**Comment by ananjaser1211** [comment]
@WoaDmulL did your phone reboot while taking the log ? because i dont see any references for that.

only thing out of the ordinary i see is you have an app com.yablio.sendfilestotv that crashes upon startup

the app crashes the systemui but im not 100% sure it crashes the entire system tho

01-16 18:53:02.515  8185  8185 E AndroidRuntime: java.lang.RuntimeException: Unable to start service com.yablio.sendfilestotv.transfer.TransferService

try to remove that app, (Send Files To TV)

**Comment by WoaDmulL** [comment]
Oh, sorry, wrong log file, I'll reupload it later. Weird enough, I don't have that app installed...it was installed when I restored a backup after flashing the ROM, but after that I uninstalled it.

**Comment by ananjaser1211** [comment]
@WoaDmulL Please try the rom without restoring apps. As that's an almost guaranteed way to get problems. Especially due to it being a version higher than stock

**Comment by WoaDmulL** [comment]
I'll close it since I can't reproduce it anymore. Thanks for your time!

**Comment by ananjaser1211** [comment]
@WoaDmulL anything in particular changed ? or they just disappeared

**Comment by WoaDmulL** [comment]
The apps won't freeze anymore and I haven't noticed any reboots since after I've started logging it. Maybe the crash reboot didn't remove the service ( Send Files To TV app ) from the startup and after I've rebooted the phone manually once it removed that entry since the app didn't exist anymore. So maybe that app was crashing the system and somehow it didn't manange to do a proper checkup of those services at the startup. Just a hunch.
